dev/bots/analyze-sample-code.dart

```diff
@@ -310,8 +310,8 @@ class SampleChecker {
     final File outputFile = File(path.join(_tempDirectory.path, '$snippetId.dart'));
     final List<String> args = <String>[
       '--output=${outputFile.absolute.path}',
-      '--input=${inputFile.absolute.path}',
-    ]..addAll(snippet.args);
+      '--input=${inputFile.absolute.path}', ...snippet.args,
```

Comment:
If this is our quick assists' edit, +1 for adding the trailing comma. We may want to also put the spread collections `'...snippet.args,'` on the next line.

@bwilkerson

Reply:
The fixes / assists do perform some minimal kinds of formatting, but don't attempt to be true to `dartfmt`. One of the things we've wanted for a long time is the ability to run `dartfmt` over a snippet of code. If we could get that, then we could generalize the fix / assist handling to make small edits, then run the formatter over the changed regions.

Reply:
> One of the things we've wanted for a long time is the ability to run dartfmt over a snippet of code

except, in this case we'd want `flutterfmt`... 😛